### PR TITLE
Admin Generator Future: Add grid config prop `toolbar` to make Toolbar optional

### PR DESCRIPTION
**Pull request by andrearutrecht**
---

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: SVK-254
-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>
  
Screenshot 2024-05-08 at 15 20 00

</details>

### CODE REVIEW

File: packages/admin/cms-admin/src/generator/future/generator.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     copyPaste?: boolean;
     readOnly?: boolean;
     filterProp?: boolean;
+    renderToolbar?: boolean;
```
Comment (nsams) [inline packages/admin/cms-admin/src/generator/future/generator.ts:62]: other options also don't have a render prefix, why not just `toolbar`?

**Comment by johnnyomair** [comment]
For forms we never render the toolbar inside the form (due to technical reasons). Should we do the same for grids?

File: packages/admin/cms-admin/src/generator/future/generator.ts (hunk 1 above)
Comment (andrearutrecht) [reply packages/admin/cms-admin/src/generator/future/generator.ts:62]: I thought `renderToolbar` would be more telling of what the option does. I will rename it to match the others

**Comment by johnnyomair** [comment]
@andrearutrecht please resolve conflicts

**Review by nsams — APPROVED** [review]

**Pull request merged**